backend/src/database/migrations/20220915123536-create-email-messages-tx.js

```diff
@@ -0,0 +1,87 @@
+'use strict'
+
+const { TransactionalEmailMessageStatus } = require('@email/models')
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('email_messages_tx', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.BIGINT,
+      },
+      user_email: {
+        type: Sequelize.DataTypes.TEXT,
+        allowNull: false,
+        references: {
+          model: 'users',
+          key: 'email',
+        },
+        onUpdate: 'CASCADE',
+      },
+      from_name: {
+        allowNull: false,
+        type: Sequelize.DataTypes.STRING(255),
+      },
+      from_address: {
+        allowNull: false,
+        type: Sequelize.DataTypes.STRING(255),
+      },
+      recipient: {
+        allowNull: false,
+        type: Sequelize.DataTypes.STRING(255),
+      },
+      params: {
+        type: Sequelize.DataTypes.JSON,
+        allowNull: false,
+      },
+      has_attachment: {
+        type: Sequelize.DataTypes.BOOLEAN,
+        allowNull: false,
+      },
+      attachment_s3_object: {
+        type: Sequelize.DataTypes.JSON,
+        allowNull: true,
+      },
+      status: {
+        type: Sequelize.DataTypes.ENUM(
+          TransactionalEmailMessageStatus.Unsent,
+          TransactionalEmailMessageStatus.InvalidFromAddressError,
+          TransactionalEmailMessageStatus.RateLimitError,
+          TransactionalEmailMessageStatus.InvalidMessageError,
+          TransactionalEmailMessageStatus.UnsupportedFileTypeError,
+          TransactionalEmailMessageStatus.MaliciousFileError,
+          TransactionalEmailMessageStatus.BlacklistedRecipientError,
+          TransactionalEmailMessageStatus.Accepted,
+          TransactionalEmailMessageStatus.Sent
```

Comment:
compared to previous db migrations, using TypeScript in a `.js` file here lowers the risk of errors

Reply:
have replaced with `...Object.values(TransactionalEmailMessageStatus)`, which is even pithier